Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -42,17 +42,17 @@ ProjectPamphletViewModelOutputs {
 
   // swiftlint:disable:next function_body_length
   public init() {
-    let configData = Signal.combineLatest(
+    let configDataAndShouldPrefix = Signal.combineLatest(
       self.configDataProperty.signal.skipNil(),
-      Signal.merge(self.viewDidLoadProperty.signal,
-                   self.viewDidAppearAnimated.signal.filter(isTrue).ignoreValues()
+      Signal.merge(self.viewDidLoadProperty.signal.mapConst(true),
+                   self.viewDidAppearAnimated.signal.filter(isTrue).mapConst(false)
```

Comment:
I hear what you're saying, but I don't really see these as being different. The only thing we'd be doing is getting rid of the `let config` and moving the whole thing to `freshProjectAndLiveStreamsAndRefTag`. In fact, I feel safer with the `combineLatest` then the `takePairWhen` to be honest. We very often put config data into a let, not sure what makes this one different?